osm2streets/src/lanes/mod.rs

```diff
@@ -271,20 +271,20 @@ impl LaneSpec {
             // https://www.gov.uk/government/publications/cycle-infrastructure-design-ltn-120 table
```

Comment:
These comments get out of date, may as well delete them. We keep talking about it, but I think a relatively easy next step is to

1) Generalize `driving_side` to return the country code anywhere on earth
2) Turn this function into data-driven config, and allow per-country overrides

Reply:
https://github.com/a-b-street/osm2lanes/pull/266 starts this process. I can get us to the point where we have a locale code easily in this repo, and we can figure out how to make use of it